pkg/knative/ingress_rule.go

```diff
@@ -39,15 +39,9 @@ func Domains(rule v1alpha1.IngressRule) []string {
 	return domains
 }
 
-func RuleIsExternal(rule v1alpha1.IngressRule, ingressVisibility v1alpha1.IngressVisibility) bool {
-	switch rule.Visibility {
-	case v1alpha1.IngressVisibilityExternalIP:
-		return true
-	case v1alpha1.IngressVisibilityClusterLocal:
+func RuleIsExternal(rule v1alpha1.IngressRule) bool {
+	if rule.Visibility == v1alpha1.IngressVisibilityClusterLocal {
```

Comment:
If I understood correctly, rule.Visibility is always set now, so this can be simplified to `rule.Visibility == IngressVisibilityExternalIP`

Reply:
Yes, that is better. Thank you.